client/app/components/ApplicationArea/ApplicationLayout/DesktopNavbar.jsx

```diff
@@ -2,13 +2,21 @@ import { first } from "lodash";
 import React, { useState } from "react";
 import Button from "antd/lib/button";
 import Menu from "antd/lib/menu";
-import Icon from "antd/lib/icon";
 import HelpTrigger from "@/components/HelpTrigger";
 import CreateDashboardDialog from "@/components/dashboards/CreateDashboardDialog";
 import { Auth, currentUser } from "@/services/auth";
 import settingsMenu from "@/services/settingsMenu";
 import logoUrl from "@/assets/images/redash_icon_small.png";
 
+import DesktopOutlined from "@ant-design/icons/DesktopOutlined";
```

Comment:
WDYT about adding an `Icon` suffix? e.g. `DesktopOutlinedIcon`

Reply:
Yep, this is a good idea, since it's better to associate the element as an Icon 👍